qa/rpc-tests/sc_cert_getblocktemplate.py

```diff
@@ -183,8 +183,8 @@ def run_test_with_scversion(self, scversion, ceasable = True):
         ### 3: Sending a transaction and a certificate
         # Generate proof before the call of `getblocktemplate`.
         # It is a time consuming operation, so may take more than GET_BLOCK_TEMPLATE_DELAY seconds.
+        quality = quality + 1
```

Comment:
Can be rewritten with `quality += 1` (coherently with line #279)

Reply:
Wouldn't  `++quality;` be even better ?

Reply:
Agree with @JackPiri , updated in dfeb52fab30e3b941b44631be6cdf8a95f317a67
Python does not have a unary increment operator, hence `++quality` and `quality += 1` are not equivalent.

Reply:
Sorry my fault. Didn't notice it was a py file.